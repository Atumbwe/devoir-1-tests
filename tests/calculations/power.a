6 6 pwr