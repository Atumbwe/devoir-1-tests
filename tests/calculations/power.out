 -------------
 <  46656  >
 -------------
        \   ^__^
         \  (oo)\_______
            (__)\       )\/\
                ||----w |
                ||     ||